pipelines/cellranger/cellranger.wdl

```diff
@@ -0,0 +1,112 @@
+workflow cellranger {
+    meta {
+        description: "Analyze 3' single-cell RNA-seq data using the 10X Genomics Cellranger pipeline."
+    }
+
+    String sample_id
+    Array[File] fastqs
+    String reference_name
+    File transcriptome_tar_gz
+    Int? expect_cells
+
+    # Runtime attributes
+    String memory = "416 GB"
+    Int boot_disk_size_gb = 12
+    String disk_space = "250"
+    Int cpu = 64
+
+    parameter_meta {
+        sample_id: "Name of sample to run CellRanger count on"
+        fastqs: "Array of fastq files for running decoupled"
+        reference_name: "Reference name for count"
+        transcriptome_tar_gz: "Cellranger transcriptome reference"
+        expect_cells: "Expected number of recovered cells (defaults to 3000)"
+        memory: "The minimum amount of RAM to use"
+        boot_disk_size_gb: "Size of disk (GB) where the docker image is booted"
+        disk_space: "Amount of disk space (GB) to allocate to the VM"
+        cpu: "The minimum number of cores to use"
+    }
+
+    call CellRanger {
+        input:
+        sample_id = sample_id,
+        fastqs = fastqs,
```

Comment:
I think we're inheriting the restrictions on cellranger fastq naming and organization, right? We might want to make a little note somewhere.